workinggroups.md

```diff
@@ -0,0 +1,14 @@
+# Open Data Cube Working Groups
+## Community Engagement Working Group (CEWG)
+
+### Description:
+
+The Community Engagement Working Group focuses on broad discussion and implementation of community engagement activities. These areas include general networking, opportunities for growing connections within the community, and an open forum for new ideas and suggestions to bolster the Open Data Cube and those who support it. 
+
+Please reach out to either chair for more information or to request a link for participating in our Monthly meetings that typically happen the last Thursday of each month at 7pm (UTC -4),  10am (UTC +11).
+
+### Current Chairs as of October 2023:
```

Comment:
Not required now, but maybe list previous chairs once the leadership is handed to another.

Reply:
Acknowledged